packages/graphback-datasync/tests/DataSyncConflictProviderTest.ts

```diff
@@ -4,6 +4,31 @@ import { ConflictError, DataSyncFieldNames, ServerSideWins, ConflictResolutionSt
 import { Context, createTestingContext } from './__util__';
 import { MAX_RETRIES } from '../src/providers/DataSyncConflictProvider';
 
+test('MAX_RETRIES has a value of 3', async () => {
+  expect(MAX_RETRIES).toBe(3);
+})
+
+describe('check configuration of MAX_RETRIES via env vars', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules() 
+    process.env = { ...OLD_ENV };
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV; 
+  });
+
+  test('will receive process.env variables', () => {
+  
+    process.env.CONFLICT_RESOLUTION_MAX_RETRIES = 3;
```

Comment:
Use a number other than 3, since this is the default value anyway it can lead to a false positive.

Reply:
Had a doubt here. TS shows a problem saying: ```Type '4' is not assignable to type 'string'.``` How do I make ```process.env.CONFLICT_RESOLUTION_MAX_RETRIES``` to not be of type string in the first place?

Reply:
Try it out with string:

```
process.env.CONFLICT_RESOLUTION_MAX_RETRIES = '4';
```